src/discord.ts

```diff
@@ -0,0 +1,73 @@
+import { Client } from "discord.js";
+import { onSelector } from "./store";
+import { createSelector } from "@reduxjs/toolkit";
+import { getLinkedAccounts } from "./store/link";
+import { getParticipants, getHasSeenStart, getCallIsActive } from "./store/zoom";
+import { mapToArray } from "./helpers";
+import { DISCORD_ACTIVE_ROLE } from "./config";
+
+
+const getActiveDiscordUsers = createSelector(
```

Comment:
Can we rename this to something more clear, like `getDiscordUsersActiveOnZoom` (that's quite a bit of a mouthful, so maybe come up with something better, but I couldn't figure out what the current name meant without reading the function--it sounded like users that were currently active on Discord).

Reply:
Thoughts on this? It seems like it would be ideal to have consistent terminology, so maybe `getOnlineDiscordUsers`? Also happy to name it something else if you think there's a better option.